api/core/core.proto

```diff
@@ -343,3 +358,68 @@ message IsCRDAvailableRequest {
 message IsCRDAvailableResponse {
     map<string,bool> clusters = 1;
 }
+
+message ListPoliciesRequest {
+    string clusterName = 1;
+    Pagination pagination = 2;
+}
+
+message ListPoliciesResponse {
+    repeated Policy policies = 1;
+    int32 total = 2;
+    string nextPageToken = 3;
+    repeated ListError errors = 4;
+}
+
+message GetPolicyRequest {
+    string policy_name = 1;
+    string clusterName = 2;
+}
+
+message GetPolicyResponse {
+    Policy policy = 1;
+    string clusterName = 2;
+}
+
+message Policy {
+    string name = 1;
+    string id = 2;
+    string code = 3;
+    string description = 4;
+    string how_to_solve = 5;
+    string category = 6;
+    repeated string tags = 7;
+    string severity = 8;
+    repeated PolicyStandard standards = 9;
+    string git_commit = 10;
+    repeated PolicyParam parameters = 11;
+    PolicyTargets targets = 12;
+    string created_at = 13;
+    string clusterName = 14;
+    string tenant = 15;
+    repeated string modes = 16;
+}
+
+message PolicyStandard {
+    string id = 1;
+    repeated string controls = 2;
+}
+
+message PolicyParam {
+    string name = 1;
+    string type = 2;
+    // value is a generic value that can be a string, int, bool and array of
+    // strings
+    google.protobuf.Any value = 3;
+    bool required = 4;
+}
+
+message PolicyTargets {
+    repeated string kinds = 1;
+    repeated PolicyTargetLabel labels = 2;
+    repeated string namespaces = 3;
+}
+
+message PolicyTargetLabel { map<string, string> values = 1; }
+
+message PolicyParamRepeatedString { repeated string value = 1; }
```

Comment:
add new line at end of file and we can use align in vs code https://marketplace.visualstudio.com/items?itemName=Chouzz.vscode-better-align for the new fields